### PR TITLE
Add missing type field  and type filter to content.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

**Comment by cb372** [comment]
:+1:

**Comment by mchv** [comment]
:+1:

File: src/main/scala/com.gu.contentapi.client/model/Queries.scala [hunk 1]
```diff
@@ -142,6 +142,7 @@ trait FilterExtendedParameters[Owner <: Parameters[Owner]] extends Parameters[Ow
   def leadContent = StringParameter("lead-content")
   def fromDate = DateParameter("from-date")
   def toDate = DateParameter("to-date")
+  def contentType = StringParameter("type")
```
Comment (mchv) [inline src/main/scala/com.gu.contentapi.client/model/Queries.scala]: I am note sure it makes a lot of sense to have the parameter on `ItemQuery` but that the same for `from-date` and `to-date` so it is a minor issue that should be addressed separately.

**Comment by mchv** [comment]
:+1:  merge it :)

**Pull request merged**